core/tests/test_utils.py

```diff
@@ -144,12 +163,27 @@ class TestBase(unittest.TestCase):
     }
 
     def _get_unicode_test_string(self, suffix):
+        """Tests if unicode string are working properly.
+
+        Args:
+            suffix: str. the suffix pattern to the UNICODE_TEST_STRING.
+
+        Returns:
+            str. string with UNICODE_TEST_STRING appended to the value
```

Comment:
str. A string that contains unicode characters and ends with the given suffix.

Reply:
Done!